### PR TITLE
[Web Portal] New UI for add and edit user

### PR DESCRIPTION
**Pull request by mslichao**
#2727 Update other UI (create/edit) for user management.

This is part of #2078 As a teacher, I need able to Batch create/update/delete user accounts for my class.

### CODE REVIEW

File: src/webportal/src/app/user/fabric/userView/BatchVirtualClustersEditor.jsx [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+  const vcsOptions = allVCs.map((vc) => {
+    return {key: vc, text: vc};
+  });
+
+  const {spacing} = getTheme();
+
+  return (
+    <Modal
+      isOpen={isOpen}
+      containerClassName={mergeStyles({maxWidth: '430px'}, t.w90)}
```
Comment (sunqinzheng) [inline src/webportal/src/app/user/fabric/userView/BatchVirtualClustersEditor.jsx]: I think you'd better add `isBlocking` property if you want to disable the light dismiss feature

File: src/webportal/src/app/user/fabric/userView/BatchPasswordEditor.jsx [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+  };
+
+  const tdPaddingStyle = c(t.pa3);
+  const tdLabelStyle = c(tdPaddingStyle, t.tr);
+
+  const {spacing} = getTheme();
+
+  return (
+    <Modal
+      isOpen={isOpen}
```
Comment (sunqinzheng) [inline src/webportal/src/app/user/fabric/userView/BatchPasswordEditor.jsx]: I think you'd better add `isBlocking` property if you want to disable the light dismiss feature

File: src/webportal/src/app/user/fabric/userView/BatchVirtualClustersEditor.jsx (hunk 1 above)
Comment (mslichao) [reply src/webportal/src/app/user/fabric/userView/BatchVirtualClustersEditor.jsx]: Good suggestion, fixed.

File: src/webportal/src/app/user/fabric/userView/BatchPasswordEditor.jsx (hunk 2 above)
Comment (mslichao) [reply src/webportal/src/app/user/fabric/userView/BatchPasswordEditor.jsx]: Good suggestion, fixed.

**Pull request merged**